### PR TITLE
Allow Fetching Posts by School

### PR DESCRIPTION
**Pull request by Leoism**
+ Allow retrieval of posts from a GET request
+ Create Tests for Fetching method
+ Require users to select a school upon making a post

### CODE REVIEW

File: capstone_project/src/test/java/com/google/sps/FetchSchoolPostsTest.java [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+import org.junit.runners.JUnit4;
+import org.mockito.Mockito;
+
+@RunWith(JUnit4.class)
+public final class FetchSchoolPostsTest extends Mockito {
+  private DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+  private HttpServletRequest request;
+  private HttpServletResponse response;
+  private LocalServiceTestHelper serviceHelper =
+      new LocalServiceTestHelper(
```
Comment (rtnavarr) [inline capstone_project/src/test/java/com/google/sps/FetchSchoolPostsTest.java:47]: nit: is the indentation supposed to be 4 spaces here? It looks like you have 6

Comment (Leoism) [reply capstone_project/src/test/java/com/google/sps/FetchSchoolPostsTest.java:47]: That fails the linter

Comment (rtnavarr) [reply capstone_project/src/test/java/com/google/sps/FetchSchoolPostsTest.java:47]: @Leoism oh ok my bad, feel free to change it back!

Comment (rtnavarr) [reply capstone_project/src/test/java/com/google/sps/FetchSchoolPostsTest.java:47]: also after you change it back, lgtm

Comment (Leoism) [reply capstone_project/src/test/java/com/google/sps/FetchSchoolPostsTest.java:47]: it's fine

**Comment by rtnavarr** [comment]
lgtm

**Review by andrewnguyen — APPROVED** [review]

**Pull request merged**